tests/datasource/fluent/test_config.py

```diff
@@ -402,15 +497,23 @@ def test_general_splitter_errors(
 
 @pytest.fixture
 @functools.lru_cache(maxsize=1)
-def from_dict_gx_config() -> GxConfig:
+def from_simple_dict_gx_config() -> GxConfig:
+    gx_config = GxConfig.parse_obj(SIMPLE_DS_DICT)
+    assert gx_config
+    return gx_config
+
+
+@pytest.fixture
+@functools.lru_cache(maxsize=1)
+def from_complex_dict_gx_config() -> GxConfig:
```

Comment:
Can you remove these `from_complex_dict_gx_config` vs `from_simple_dict_gx_config` extra fixtures.
We gain nothing from having a `from_simple_dict_gx_config` fixture

Reply:
@Kilo59 I deleted  `from_simple_dict_gx_config` fixture -- it is not used now that `assets` removal is not part of this change; good catch.  Thanks.